docs/distros/macos.md

```diff
@@ -0,0 +1,38 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Brew tap for macOS
+
+If you're using macOS and `brew`, you can install Tezos binaries from the taps provided
+by this repository. There are two taps: one for the latest stable release of Tezos and
+one for the latest release candidate of Tezos.
+
+In order to use latest stable version run the following:
+```
+brew tap serokell/tezos-packaging-stable https://github.com/serokell/tezos-packaging-stable.git
+```
+
+In order to use latest release candidate version run the following:
+```
+brew tap serokell/tezos-packaging-rc https://github.com/serokell/tezos-packaging-rc.git
+```
+
+Once the desired tap is selected, you can install the chosen package, e.g.:
+```
+brew install tezos-client
+```
+
+## Building brew bottles
+
+It's possible to provide prebuilt macOS packages for brew called bottles. They're supposed
+to be built before making the new release and included to it. In order to build all bottles run
+`build-bottles.sh` script:
+```
+./scripts/build-bottles.sh
+```
+
+Note that this might take a while, because builds don't share common parts and for each binary
+dependencies are compiled from scratch. Once the bottles are built, the corresponding sections in the
+formulas should be updated. Also, bottles should be uploaded to the release artifacts.
```

Comment:
I'd reword this a little bit, this is mainly going to be seen by users, who aren't really interested in this part.

I'd change it to first mention that bottles are provided with every release for faster installation and then only mention that if one wants to build them there is a script to do so.